docs/API.md

```diff
@@ -697,6 +697,23 @@ query HelloWorld {
 | tcp.remote.host    | Remote hostname attempting to connection with |
 | tcp.remote.port    | Remote port attempting to connect to          |
 
+#### Nest
+
+##### Tags
+
+| Tag                      | Description                                               |
+|--------------------------|-----------------------------------------------------------|
+| request.url              | The complete URL of the request.                          |
```

Comment:
`http.url` and 'http.method` per OT/OTel

Reply:
Also do you mind adding the `component` tag that's set?

Reply:
Done